src/main/java/com/exactpro/th2/infraoperator/operator/manager/impl/DefaultWatchManager.java

```diff
@@ -557,6 +562,19 @@ public void onClose(KubernetesClientException cause) {
             if (cause != null)
                 logger.error("Watcher closed ({})", this.getClass().getSimpleName(), cause);
         }
+
+        private <T extends Identifiable> void checkForSameName(List<T> links, String annotation) {
```

Comment:
dublicates can be detected more easely and effectively using Set-s instead of streams